### ISSUE TITLE
Laradock Nginx: This site can’t be reached

### ISSUE THREAD

**Issue by GinoPane** [laradock/laradock#1405]
### Info:
- Docker version (`$ docker --version`): 17.12.0-ce, build c97c6d6
- Laradock commit (`$ git rev-parse HEAD`): 0077de2
- System info (Mac, PC, Linux): Windows 10 Enterprise
- System info disto/version: 1709

### Issue:
I'm stuck with `This site can’t be reached` error from my local docker container running Nginx from the latest [Laradock][1] installation.

I'm running Docker version 17.12.0-ce, build c97c6d6 on Windows 10 Enterprise (1709).

Everything worked fine a couple months before and today I wanted to run my sandbox project and faced this issue.

I've checked that `mysite.conf` is listed in "sites-available" directory of Nginx container.

Also I see that there are `mysite_access.log` and `mysite_error.log` in `/var/log` (both empty, because I tried to rename the site just to check if it helps).

Also, my test host is listed in `hosts` file:

	127.0.0.1	mysite.loc

The same server name is in `mysite.conf` file.

And still I have `This site can’t be reached` error when I try to open mysite.loc in the browser (Chrome Version 64.0.3282.186 (Official Build) (64-bit)). Simple `localhost` is not accessible too.

Here's what `docker ps` gives me (ports are normally exposed):

[![enter image description here][2]][2]

Does anyone have an idea what else should I check or fix to get this running? Am I missing anything? 

  [1]: https://github.com/laradock/laradock/
  [2]: https://i.stack.imgur.com/ACH5a.png

Issue is described here - https://stackoverflow.com/questions/49017135/laradock-nginx-this-site-can-t-be-reached. 
_____

### Expected behavior:
The site should be available when Nginx is running.

**Comment by bestlong** [comment]
@GinoPane Please show your mysite.conf content and other config change.

if hostname not match any nginx conf `server_name` will use default config.

**Comment by GinoPane** [comment]
@bestlong Today I've found out that there was a bug introduced by one of the previous updates when a new port forwarding for Windows was implemented: https://github.com/docker/for-win/issues/1506.

And it looks like that it was my issue. I've upgraded to the latest version (Docker version 17.12.0-ce, build c97c6d6) and now my dev host is reachable (without any changes anywhere else).

Anyway, thanks for looking into this!
